packages/docusaurus/src/server/configValidation.ts

```diff
@@ -214,6 +216,18 @@ export const ConfigSchema = Joi.object<DocusaurusConfig>({
     })
     .default(DEFAULT_CONFIG.scripts),
   ssrTemplate: Joi.string(),
+  headTags: Joi.array()
+    .items(
+      Joi.object({
+        tagName: Joi.string().required(),
+        attributes: Joi.object().required(),
```

Comment:
wonder if we should validate that object values should be strings? Is there any case where it's something else?

Also we'd want to have tests including rejections to ensure error messages are clear

Reply:
I couldn't work out where tests should be - can you point me in the right direction?